### PR TITLE
Add mixin for IBM DB2 integration

### PR DESCRIPTION
**Pull request by dpaasman00**
This mixin adds 1 dashboard and 3 alerts.

Alerts:
- IBMDB2HighLockWaitTime
- IBMDB2HighNumberOfDeadlocks
- IBMDB2LogUsageReachingLimit

Screenshot 2023-06-19 at 2 28 49 PM
Screenshot 2023-06-19 at 2 28 58 PM

### CODE REVIEW

**Review by kuiperda — APPROVED** [review]

**Review by schmikei — APPROVED** [review]

File: mixin/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        alertsLogUsageReachingLimit: 90, //percent 0-100
+    },
+}
+```
+
+## IBM DB2 overview
+The IBM DB2 overview dashboard provides details about the general state of the database like bufferpool hit ratio, active connections, and deadlocks.
+
+![First screenshot of the overview dashboard]()
+![Second screenshot of the overview dashboard]()
```
Comment (rgeyer) [inline mixin/README.md:32]:
```suggestion
![First screenshot of the overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/ibm-db2/screenshots/ibm-db2-overview-1.png)
![Second screenshot of the overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/ibm-db2/screenshots/ibm-db2-overview-2.png)
```

**Review by rgeyer — APPROVED** [review]
LGTM. Just added the screenshot URLs now that they've been uploaded.

File: mixin/dashboards/ibm-db2-overview.libsonnet [hunk 2]
```diff
@@ -0,0 +1,874 @@
...
+        uid=dashboardUid,
+      )
+
+      .addTemplates(
+        std.flattenArrays([
+          [
+            template.datasource(
+              promDatasourceName,
+              'prometheus',
+              null,
```
Comment (v-zhuravlev) [inline mixin/dashboards/ibm-db2-overview.libsonnet:800]: Normally, our mixins/integration have datasource variables available (visible) to users. I think it is better to show loki and prometheus datasources as variables on top.

Also, by convention, if you have two datasources(metrics + logs) on the same dashboards in variables, they should be named:
- prometheus_datasource
- loki_datasource

**Review by v-zhuravlev — CHANGES_REQUESTED** [review]
Thanks for this mixin! Added one suggestion to the dashboard.

File: mixin/dashboards/ibm-db2-overview.libsonnet (hunk 2 above)
Comment (dpaasman00) [reply mixin/dashboards/ibm-db2-overview.libsonnet:800]: Hey @v-zhuravlev, so I looked into your comment and came up with a few things. 

- I believe the datasource selectors are present currently. When I run "make dashboards_out" and import the resulting json file into the shared cloud instance the datasource selectors appear at the top. If you wanna check it out, it should be under the "Integration - IBM DB2" dashboard folder in the shared cloud instance. 
- I'm also pretty sure that the variables are named "prometheus_datasource" and "loki_datasource". On line 798 theres a ```promDatasourceName``` referenced, and on line 9 it assigns that variable with "prometheus_datasource". So I think that's how it's naming the prometheus data source, which is how I saw it named using the json file generated by dashboards_out. A similar thing occurs with logs (lines 807 and 10).
- Not sure if this is what caught your eye, but the screenshots I've included are lacking the datasource selectors. I can get some better screenshots and send them your guy's way and have them replace the pictures that've been uploaded already.

Let me know your thoughts and if I missed anything! I think I addressed your concern with the datasource selectors, but please let me know if I missed anything. Thanks!

Comment (v-zhuravlev) [reply mixin/dashboards/ibm-db2-overview.libsonnet:800]: You are right, I think I got confused by the screenshot. Sorry about that!

**Review by v-zhuravlev — APPROVED** [review]

**Pull request merged**